brianlong/www.validators.app
# 181243773 ping thing daemon

#### What's this PR do?
- add daemon for ping things instead of worker

#### How should this be manually tested?
- run daemon/process_ping_thing_daemon.rb and create some PingTHingRaws.
- Check if it's processed into Ping Things

#### What are the relevant tickets?
- [URL to the PivotalTracker ticket](https://www.pivotaltracker.com/story/show/181243773)

#### Task completed checklist:
- [] Is there appropriate test coverage?
- [] Did I take into consideration possible security vulnerabilities?
- [] Are the controllers secure?
- [] Was every important matter documented?

#### Things to watch out for
- [N] Does this PR have any migrations?
- [N] Does this PR add new dependencies?
